docs/md/docs/2020-07-09-last-dance.md

```diff
@@ -0,0 +1,96 @@
+* Jerry Reinsdorf and the rest of the Chicago Bulls mgmt - different stakeholders, "upper management"
+* The fans and the crowd - customers
+
+Now that we have this comparison in mind we can begin with the real fun :)
+
+## The team lead's approach to winning
+The coach/team lead has so much on their heads:
+* Satisfying all stakeholders inside the organization - bringing money in by winning/selling
+* Satisfying the fans/customers in the long term
+* Satisfying the team itself - personal growth, ego, salaries
+* Satisfying themselves - let's not forget that the coach/team lead wants to develop and enjoy as well
+<br>
+Let's compare the two approaches of the two coaches in the series:
+
+### Doug Collins - win by maximizing reliance on Michael Jordan
+Doug Collins was a coach that brought great results to the team by putting all the focus on Michael Jordan, all the game moves and tactics was built on Michael Jordan's abilities.
+This is an example to a team lead that puts the weight of the team's ability to deliver only on the senior engineers in the team:
+* Tough and complicated tasks are assigned to the senior engineers
+* The non-senior engineers' purpose is to let the senior engineers run fast and deliver
+
+This made the Chicago Bulls a great team because they had one hell of a senior engineer :) but they could not win the championship.
+There are a lot of disadvantages to this approach:
+* The team's ability to deliver is equal to the senior engineers' ability to deliver
+* The ability to improve is equal to the ability of your specific senior engineers to improve
+* This is not scalable and relies too much on too few people in your team
+* The non-senior team members do not improve, this approach perpetuates the current "snapshot" of the team - the seniors will stay seniors, the non-seniors will not be promoted.
+
+So even though the results were good, they were not great. Remember all the concerns team leads have on their heads? this approach led to unsatisfied internal stakeholders (not winning the championship), unstaisfied fans/customers (not winning the championship => the product quality was limited and did not improve), unsatisfied team (they were all a shadow of the senior engineer with no future to be seniors themselves), which makes the team lead itself unsatisfied - and who can blame them, how can you be satisfied when everyone around you are not?
+
+### Phil Jackson - win by maximizing the team performance
+Phil was hard to process for the team, he arrived with a completely different approach which basically turned upside down the way the team behaved so far. <br>
+The discussions and plans moved from "what can we do to make Michael Jordan score?" to "what can we do to win in the long term?". This is a huge difference that made the real change in the group. <br>
+Phil put emphasize on team building, made everyone focused on the target of winning and to rely on each other. "There is no I in team". <br><br>
```

Comment:
Phil being an inclusive leader, fostering collaboration (reducing bus factor) and not necessarily hiring new team members for their resume but their ability to complement the team. letting their creativity loose